testSetup.js

```diff
@@ -0,0 +1,3 @@
+const matchers = require("jest-extended");
```

Comment:
This isn't strictly necessary and would have been fine in your setupFilesAfterEnv array.